.travis/cli-tests.sh

```diff
@@ -49,11 +49,13 @@ echo
 echo $WHITE • harness/nnef-test-cases $NC
 echo
 
+export CARGO_OPTS="--no-default-features"
```

Comment:
I'm not too sure about this: unless I'm wrong, we are likely to speed up this (small) set of test, but then we will need to build tract again for the following test as the active feature set will be different.

Reply:
Yes that's true, I'll remove it